### PR TITLE
rename pcc to gemfire, and update version to 1.0.0

### PR DESCRIPTION
**Pull request by jinmeiliao**

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -16,7 +16,7 @@
 # limitations under the License.
 
 ./build.sh
-echo Y | cf install-plugin pcc
+echo Y | cf install-plugin gemfire
 echo "    _______  ________    ________  _______  _______"
 echo "   / _____/ / ______/   / ____  / / _____/ / _____/"
```
Comment (dschneider-pivotal) [inline install.sh]: these "echo" lines print CF PCC in giant letters

File: reinstall.sh [hunk 2]
```diff
@@ -15,9 +15,9 @@
...
 # limitations under the License.
 
-cf uninstall-plugin pcc
-go build -o ./pcc ./cmd/main
-echo Y | cf install-plugin pcc
+cf uninstall-plugin gemfire
+go build -o ./gemfire ./cmd/main
+echo Y | cf install-plugin gemfire
 
 echo "    _______  ________    ________  _______  _______"
```
Comment (dschneider-pivotal) [inline reinstall.sh]: another giant letters CF PCC

File: impl/pcc/plugin.go [hunk 3]
```diff
@@ -78,7 +78,7 @@ func (c *BasicPlugin) Run(cliConnection plugin.CliConnection, args []string) {
 // GetMetadata provides metadata about the CF plugin including a helptext for the user
 func (c *BasicPlugin) GetMetadata() plugin.PluginMetadata {
 	return plugin.PluginMetadata{
-		Name:    "pcc",
+		Name:    "gemfire",
```
Comment (dschneider-pivotal) [inline impl/pcc/plugin.go]: Note that this go file is in "impl/pcc/". Should it be changed to "impl/gemfire"?

File: install.sh (hunk 1 above)
Comment (dschneider-pivotal) [reply install.sh]: Also note the hard coded "1.0.0" in this echo. Would it be possible to read these numbers at runtime from VersionType you updated on structs.go?

File: impl/pcc/plugin.go (hunk 3 above)
Comment (jmelchio) [reply impl/pcc/plugin.go]: Yeah, it probably makes sense to rename the packages as well.

Comment (jmelchio) [reply impl/pcc/plugin.go]: Yes, this should be changed.

File: install.sh (hunk 1 above)
Comment (jmelchio) [reply install.sh]: Yeah, makes sense to get them from there.

**Pull request merged**